pom.xml

```diff
@@ -23,10 +23,15 @@
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
 <!--		uncomment while you add database details-->
-<!--		<dependency>-->
-<!--			<groupId>org.springframework.boot</groupId>-->
-<!--			<artifactId>spring-boot-starter-data-jpa</artifactId>-->
-<!--		</dependency>-->
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-jpa</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>javax.validation</groupId>
+			<artifactId>validation-api</artifactId>
```

Comment:
Lets not add validations like this.. you will have to error ha dling for them using complex way